python/mooseutils/tools.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+import os
+import traceback
+from mooseutils import colorText
+
+try:
+    from PyQt5 import QtWidgets, QtCore
+    MOOSE_USE_QT5 = True
+except:
+    MOOSE_USE_QT5 = False
+
+"""
+Global for enabling/disabling debug mode.
+"""
+MOOSE_DEBUG_MODE = False
```

Comment:
I also think that this should be renamed to message.py and pull out the gold, touch, unique_list stuff and put that in mooseutils.py